.changeset/tough-olives-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-docs/writing-style': patch
+---
+
+Applies all suggested writing style improvements.
```

Comment:
Please list the actual changes in minimal form. Otherwise is hard to reconcile what to adjust in the content when the release is applied.